doc/api/dgram.md

```diff
@@ -461,6 +461,23 @@ added: v8.7.0
 
 This method throws [`ERR_SOCKET_BUFFER_SIZE`][] if called on an unbound socket.
 
+### `socket.getSendQueueSize()`
```

Comment:
I am wondering if these should be read-only properties (i.e., properties without setters) instead of functions. Unless that would be inconsistent with other APIs, or maybe there's a chance we'll want to pass arguments in the future?

Reply:
I think this should be read-only property. Here is just to be consistent with the previous APIs, such as `getRecvBufferSize`, `getSendBufferSize`.